src/seedu/addressbook/AddressBook.java

```diff
@@ -377,12 +382,14 @@ private static String executeCommand(String userInputString) {
             return executeListAllPersonsInAddressBook();
         case COMMAND_DELETE_WORD:
             return executeDeletePerson(commandArgs);
+        case COMMAND_SORT_WORD:
+        	return executeSortAllPersonsInAddressBook();
         case COMMAND_CLEAR_WORD:
             return executeClearAddressBook();
         case COMMAND_HELP_WORD:
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
-            executeExitProgramRequest();
+        	exitProgram();
```

Comment:
indentation is off here; our coding standard requires using 4 spaces for indentation. 
you can get Eclipse to do this for you, please check out how?